src/core/Akka.MultiNodeTestRunner.Shared/Sinks/TimelineLogCollectorActor.cs

```diff
@@ -46,23 +46,25 @@ public TimelineLogCollectorActor()
                         return (NodeIndex: node.Index, NodeRole: node.Role, Logs: nodeMessages.Select(m => m.ToString()).ToList());
                     }).ToList()
                 };
-                
+
                 Sender.Tell(log);
             });
-            
+
             Receive<DumpToFile>(dump =>
             {
                 // Verify that directory exists
                 var dir = new DirectoryInfo(Path.GetDirectoryName(dump.FilePath));
                 if (!dir.Exists)
                     dir.Create();
-                
+
                 File.AppendAllLines(dump.FilePath, _timeline.Select(pairs => pairs.Value).SelectMany(msg => msg).Select(m => m.ToString()));
                 Sender.Tell(Done.Instance);
             });
-            
+
             Receive<PrintToConsole>(_ =>
             {
+                LogMessageInfo.TryParseLogLevel(_.MinimumLogLevel, out var minimumLogLevel);
```

Comment:
`_` is usually used as not used argument, so we should probably rename it to something more meaningful - just to avoid people thinking that there are no parameters used

Reply:
done